src/platform/packages/shared/kbn-scout/test/scout/tests/api_services/fleet.spec.ts

```diff
@@ -0,0 +1,409 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { apiTest, expect } from '../../../../src/playwright';
+
+apiTest.describe('Fleet Integration Management', { tag: ['@svlSecurity', '@ess'] }, () => {
+  let integrationName: string;
+
+  apiTest.beforeEach(async () => {
+    integrationName = `test-integration-${Date.now()}`;
+  });
+
+  apiTest.afterEach(async ({ apiServices }) => {
+    // Clean up integration
+    try {
+      await apiServices.fleet.integration.delete(integrationName);
+    } catch (e) {
+      // Integration might not exist or already deleted
+    }
```

Comment:
💭  we probably don't want consumers to put try/catch while using our helpers. I suggest we add status code to `ignore` in the helpers code.